packages/massa-web3/src/web3/accounts/WalletProviderAccount.ts

```diff
@@ -60,15 +60,16 @@ export class WalletProviderAccount implements IBaseAccount {
       paramToSend = params;
     }
 
-    let res = await this.account.callSC(
+    let res = (await this.account.callSC(
       callData.targetAddress,
       callData.functionName,
       paramToSend,
       callData.coins,
       callData.fee,
       callData.maxGas,
       false,
-    );
+    )) as ITransactionDetails;
```

Comment:
why ? :(
`let res: ITransactionDetails = await .... ` should work. but it should be infered